### PR TITLE
frame change implementation, code clean up and turning off react warnings and end screen crash

### PR DESCRIPTION
**Pull request by wunnath**

### CODE REVIEW

File: sdk/android/app/src/main/assets/skin.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
         "color": "white"
       }
     },
-    "showCountDownTimerOnEndScreen": true,
+    "showCountDownTimerOnEndScreen": false,
```
Comment (lental) [inline sdk/android/app/src/main/assets/skin.json]: you can't change the skin.json as part of the application.  You need to change the javascript code

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java [hunk 2]
```diff
@@ -32,7 +32,14 @@
   private ReactInstanceManager _reactInstanceManager;
   private ReactRootView _rootView;
   private int viewWidth,viewHeight;
+  private listnerFrame sizeChange;
 
+  public interface listnerFrame{
```
Comment (lental) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: mispelled: "listener"

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java [hunk 3]
```diff
@@ -32,7 +32,14 @@
   private ReactInstanceManager _reactInstanceManager;
   private ReactRootView _rootView;
   private int viewWidth,viewHeight;
+  private listnerFrame sizeChange;
```
Comment (lental) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: rename class - FrameChangeListener, Rename variable - frameChangeListener

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java [hunk 4]
```diff
@@ -32,7 +32,14 @@
...
 
+  public interface listnerFrame{
+      public void onFrameChange(int width, int height);
```
Comment (lental) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: why don't you pass all 4 variables?  no reason to prune it at this level

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java (hunk 3 above)
Comment (wunnath) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: done

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java (hunk 2 above)
Comment (wunnath) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: changed it to FrameChangeListener

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java (hunk 4 above)
Comment (wunnath) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: passed the previous values also now.

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java (hunk 3 above)
Comment (lental) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: This doesn't look done.  Your changes did not get into this branch yet

Comment (wunnath) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayout.java]: please review now.

File: sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/util/ReactUtil.java [hunk 5]
```diff
@@ -21,6 +21,7 @@ public static void setJSServer(String host, Context context) {
         SharedPreferences mPreferences = PreferenceManager.getDefaultSharedPreferences(context);
         SharedPreferences.Editor editor = mPreferences.edit();
         editor.putString("debug_http_host", host);
+        editor.putBoolean("js_dev_mode_debug", false);
```
Comment (lental) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/util/ReactUtil.java]: what does this do?  does this hide errors? We don't want to hide errors at them oment

Comment (wunnath) [inline sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/util/ReactUtil.java]: removed this line and will handle the warnings on the react side itself.

**Pull request merged**